packages/next-swc/crates/next-core/src/next_app/metadata/route.rs

```diff
@@ -213,21 +242,19 @@ async fn dynamic_site_map_route_source(
     let stem = path.file_stem().await?;
     let stem = stem.as_deref().unwrap_or_default();
     let ext = &*path.extension().await?;
-
     let content_type = get_content_type(path).await?;
-
     let mut static_generation_code = "";
 
-    if mode.is_production() && page.contains(&PageSegment::Dynamic("[__metadata_id__]".into())) {
+    if mode.is_production() && page.contains(&PageSegment::Dynamic("__metadata_id__".into())) {
         static_generation_code = indoc! {
             r#"
                 export async function generateStaticParams() {
                     const sitemaps = await generateSitemaps()
                     const params = []
 
-                    for (const item of sitemaps) {
-                        params.push({ __metadata_id__: item.id.toString() })
-                    }
+                    for (const item of sitemaps) {{
+                        params.push({ __metadata_id__: item.id.toString() + '.xml' })
+                    }}
```

Comment:
```suggestion
                    for (const item of sitemaps) {
                        params.push({ __metadata_id__: item.id.toString() + '.xml' })
                    }
```

That's not in a format! function

Reply:
The `r#` requires to have it otherwise it errors
```
error: invalid format string: expected `'}'`, found `'i'`
   --> packages/next-swc/crates/next-core/src/next_app/metadata/route.rs:287:3
    |
286 |                         status: 404,
    |                          - because of this opening brace
287 |                     }})
    |   ^ expected `'}'` in format string
    |
    = note: if you intended to print `{`, you can escape it using `{{`
```